src/main/java/Controller/musicEngineController.java

```diff
@@ -115,7 +131,7 @@ private void stop(){
             case SONG:
                 break;
             case PLAYLIST:
-                playPlaylist.stopQueue();
+                this.playPlaylist.stopQueue();
```

Comment:
in what scenarios would you stop playing the playlist? we should have a reset playbar function to call here (and possibly case SONG) to set the playbar to blank when we're not playing anything.

Reply:
`stop()` is a bit of a misleading name, it doesn't immediately stop the currently playing song, it stops the queuing of future songs. thus, any time you want to play something new (eg. a song, another playlist, a space), the playlist must be stopped.